package.json

```diff
@@ -26,7 +26,8 @@
     "standard": "^10.0.0",
     "standard-markdown": "^4.0.0",
     "sumchecker": "^2.0.2",
-    "temp": "^0.8.3"
+    "temp": "^0.8.3",
+	"cross-var": "^1.1.0"
```

Comment:
Some weird indenting going on here

Reply:
I don't know what is wrong with the display, but the file seems ok to me:
![image](https://user-images.githubusercontent.com/25103534/41978510-51d0fd5e-7a22-11e8-9932-3a840fb5387c.png)

Reply:
You were right, i replaced tab by spaces and the issue is gone, thanks!